Practice2/README.md

```diff
@@ -49,6 +46,8 @@
       * version.css
     * variables/
       * main-var.css
+    * base.css
```

Comment:
Base and reset should be move to base/
It cannot have same level with main

Reply:
fixed